src/algorithm/half_node.rs

```diff
@@ -110,8 +114,10 @@ mod test {
         // N(entry) -> head -> split            > N(merge) -> N(tail) -> N(exit)
         //               |          \-> right -/                 |
         //               \---<---<---<---<---<---<---<---<---<---/
-        // Allowing to identity two nested regions (and fixing the problem with a SimpleCfgView on the same example)
+        // Allowing to identity two nested regions (and fixing the problem with an IdentityCfgMap on the same example)
```

Comment:
```suggestion
        // Allowing to identify two nested regions (and fixing the problem with an IdentityCfgMap on the same example)
```
(Not strictly part of this PR but may as well fix here.)